lib/translighterate.rb

```diff
@@ -17,7 +17,13 @@ def self.highlight(text, phrases, options = {})
 
       # Transliteration trick from:
       #  https://github.com/cubing/worldcubeassociation.org/issues/238#issuecomment-234702800
-      transliterated = text.chars.map { |ch| transliterate_char(ch) }.join
+      transliterated = text.chars.map do |ch|
+        begin
+          transliterate_char(ch)
```

Comment:
I'm pretty afraid of "pokemon exception handling".

I feel like a cleaner solution might be to change the behavior of `transliterate_char`. Can we modify it so it returns either empty string or the same string we're passing into it?

Also, it would be great to add a test of all this!

Reply:
I didn't even realize the function was defined here, I thought it was some ruby built in thing...
Definitely, I'll edit the function!